Malay-Kumar-Parida/02.Variables/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import "fmt"
+
+var l float32 = 100 //lower case Thorughout This package
+var L int = 50      //Visible everywhere
+//Cant used 3rd type of declarration here
+
+//Variables can be grouped like this
+var (
+	name    string = "Malay"
+	college string = "VSSUT"
+	regd    int    = 100
+)
+
+func main() {
+	var i int //1st style of Variable declaration
```

Comment:
```suggestion
	var i int // Variable declaration without initialization
```
what is `1st type` and `2nd type` of variable declaration. You have to be clear when you comment.

Reply:
this too